Formula/inform6.rb

```diff
@@ -20,10 +20,8 @@ class Inform6 < Formula
   end
 
   def install
-    # Disable parallel build until release with https://gitlab.com/DavidGriffith/inform6unix/-/commit/dab07d5c83a42e1c52e4058d6a31a8137f54b59c
-    # ships; see https://gitlab.com/DavidGriffith/inform6unix/-/issues/26
-    ENV.deparallelize
-    system "make", "PREFIX=#{prefix}", "MAN_PREFIX=#{man}", "install"
+    bin.mkdir
```

Comment:
Or might be better to just restore `ENV.deparallelize`.
Both seem to work, so it seems parallel install is causing issues for `bin` directory.